index.js

```diff
@@ -1,8 +1,23 @@
-const express = require('express')
-const path = require('path')
-const PORT = process.env.PORT || 5000
-const app = express()
+const express = require('express');
+const app = express();
+const path = require('path');
+const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser')
+const db = require('./server/db/db');
+const logic = require('./server/routes/logic');
+const PORT = process.env.PORT || 5000;
 
-app.use(express.static(path.join(__dirname, 'assets')))
-app.get('/', (req, res) => res.sendFile(path.join(__dirname,'assets/html/home.html')))
+app.use(bodyParser.urlencoded({ extended: false}))
+app.use(bodyParser.json())
+app.use(cookieParser())
+app.use(express.json())
+app.use(express.static(path.join(__dirname, 'client')))
+app.get('/', (req, res) => res.sendFile(path.join(__dirname,'client/html/home.html')))
```

Comment:
can you put a line above this line to differentiate between when your routes start and what middleware you use in the beginning?